### PR TITLE
LIMS-1869 Not possible to create an Analysis Request

### PR DESCRIPTION
**Pull request by Espurna**
https://jira.bikalabs.com/browse/LIMS-1869?filter=10501

### CODE REVIEW

**Comment by rockfruit** [comment]
what was the actual problem here, that the workflows needed to be reimported?

**Comment by Espurna** [comment]
Just added worflows to avoid future work :)

**Comment by rockfruit** [comment]
+1 for avoiding any kind of work :) thanks

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 3.1.9 (unreleased)
 ------------------
+LIMS-1869: Not possible to create an Analysis Request
 LIMS-1867: Auto-header, auto-footer and auto-pagination in results reports
 LIMS-1743: Reports: ISO (A4) or ANSI (letter) pdf report size
+LIMS-1695: Invoice export function missing
```
Comment (xispa) [inline docs/CHANGELOG.txt]: PR not accepted yet... https://github.com/bikalabs/Bika-LIMS/pull/1548
I am waiting for the conclusions from the discussion at JIRA.

**Pull request merged**